src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/ReadMetadata.java

```diff
@@ -361,6 +361,15 @@ public static void writeMetadata(final ReadMetadata readMetadata,
                                 "+" + stats.getPositiveMAD() + "\t" + stats.getCoverage() + "\t" +
                                 stats.getMeanBaseQuality() + "\t" +
                                 stats.getNReads() + "\t" + stats.getReadStartFrequency() + "\n");
+                final IntHistogram.CDF templateSizeCdf = stats.getCDF();
+                final int cdfSize = templateSizeCdf.size();
+                final long totalObservations = templateSizeCdf.getTotalObservations();
+                writer.write("template size cumulative counts:");
+                for(int idx = 0; idx < cdfSize; ++idx) {
+                    final long cumulativeCounts = Math.round(templateSizeCdf.getFraction(idx) * totalObservations);
```

Comment:
Why not just go by the CDF -- why multiply by the number of observations?

Reply:
Since these floats all originate as integers divided by totalObservations, doing it this way allows me to output the full precision result to text with fewer characters and no special print format overrides.